lib/ronin/web/cli/commands/diff.rb

```diff
@@ -109,25 +109,26 @@ def read(source)
 
           private
 
-          def load_doc(page1, page2)
-            case options[:format].to_sym
+          #
+          # Loads the given html or xml sources
+          #
+          # @param [String] page1
+          #   The URL or file path of the original page.
+          #
+          # @param [String] page2
+          #   The URL or file path of the modified page.
+          # @return [@Nokogiri::HTML, @Nokogiri::HTML] or [@Nokogiri::XML, @Nokogiri::XML]
```

Comment:
1. Add a newline above `@return`.
2. No need to put a `@` before `Nokogiri`.
3. To represent the type of an Array containing a fixed number of elements, use the syntax `(Type1, Type2, Type3)`. To represent multiple possible types of Arrays containing two arguments, use the syntax `@return [(Nokogiri::HTML::Document, Nokogiri::HTML::Document), (Nokogiri::XML::Document, Nokogiri::XML::Document)]`.